garminexport/incremental_backup.py

```diff
@@ -16,16 +15,15 @@
 def incremental_backup(username: str,
                        password: str = None,
                        backup_dir: str = os.path.join(".", "activities"),
-                       export_formats: List[str] = None,
+                       export_formats: str = 'ALL',
```

Comment:
This is supposed to be a `list` of `str`, so I would suggest switching the type annotation back.
We could have the default be `supported_export_formats` instead.